market/admin.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib import admin
+from market.models import Customers, Goods, Orders
+from .admin_filters import PriceCustomFilter
+from .admin_actions import change_quantity
+
+# Register your models here.
+
+
+@admin.register(Customers)
+class CustomersAdmin(admin.ModelAdmin):
+    all_fields = ('id', 'name', 'email', 'address', 'phone', 'sign_up_date')
+    list_display = all_fields
+    ordering = all_fields
+    list_filter = ('sign_up_date',)
+    list_display_links = ('name',)
+    readonly_fields = ('id', 'sign_up_date')
```

Comment:
Кортежи не требуют скобок при создании
```suggestion
    all_fields = 'id', 'name', 'email', 'address', 'phone', 'sign_up_date'
    list_display = all_fields
    ordering = all_fields
    list_filter = 'sign_up_date',
    list_display_links = 'name',
    readonly_fields = 'id', 'sign_up_date'
```